src/javascript/JContent/ContentRoute/ToolBar/Toolbar.scss

```diff
@@ -5,3 +5,7 @@
 .spacer {
     width: var(--spacing-small);
 }
+
+.selection {
+    margin-top: 5px;
```

Comment:
You should use moonstone's tokens...

Reply:
But I am also wondering if this is not a dirty hack.... :confused:
The difference of heights is likely related to buttons/texts using different sizes, doesn't it?